### DISCUSSION TITLE
Can you figure out whose birthday is on each day?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#479] [Galaxy Trivia] [answered]
George, William, John, Abe, and Millard have their birthdays on consecutive days, all between Monday and Friday. George’s birthday is as many days before Millard’s and William’s is after Abe’s. John is two days older than Abe. Millard’s birthday is on Thursday.

**Comment by peterbaccaro** [comment, marked as answer]
John’s is on Monday, George’s on Tuesday, Abe’s on Wednesday, Millard’s on Thursday, and William’s on Friday.
